judge/management/commands/export_contest_submissions.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from django.core.management.base import BaseCommand, CommandError
+
+from judge.models import Contest, ContestParticipation
+
+
+class Command(BaseCommand):
+    help = 'export contest submissions'
+
+    def add_arguments(self, parser):
+        parser.add_argument('key', help='contest key')
+        parser.add_argument('output', help='output directory')
+
+    def handle(self, *args, **options):
+        contest_key = options['key']
+        output_dir = options['output']
+
+        contest = Contest.objects.filter(key=contest_key).first()
+        if contest is None:
+            raise CommandError('contest not found')
+
+        if os.path.exists(output_dir):
+            raise CommandError('output directory already exists')
+
+        os.makedirs(output_dir)
+
+        users = contest.users.filter(virtual=ContestParticipation.LIVE).prefetch_related('submissions').all()
+
+        for user in users:
+            username = user.user.user.username
+            user_dir = os.path.join(output_dir, username)
+            user_history_dir = os.path.join(user_dir, '$History')
+
+            os.makedirs(user_dir)
+
+            problems = set()
+            submissions = user.submissions.order_by('-id') \
+                .select_related('problem__problem', 'submission__source', 'submission__language') \
+                .values_list('problem__problem__code', 'submission__source__source',
+                             'submission__language__extension', 'submission__id')
+
+            for problem, source, ext, sub_id in submissions:
+                if problem not in problems:  # Last submission
```

Comment:
This code is too "quiet". 
Maybe add a simple "process bar" here?

Also please print execution time.